astropy/io/fits/tests/test_header.py

```diff
@@ -280,7 +280,7 @@ def __getitem__(self, key):
 
         header.update(NAXIS1=100, NAXIS2=100)
         assert set(header.keys()) == {"FOO", "A", "B", "HELLO", "NAXIS1", "NAXIS2"}
-        assert set(header.values()) == {"BAR", 1, 2, 100, 100}
+        assert set(header.values()) == {"BAR", 1, 2, 100}
```

Comment:
```suggestion
        assert tuple(header.values()) == ("BAR", 1, 2, 100, 100)
```
 Might work

Reply:
I verified that the following works:
```
assert tuple(header.values()) == ("BAR", 1, 2, 1, 100, 100)
```
How ever, current "fits" package doc doesn't mention anything about ordering of the items. Is it ok to assume it? Also, Python dict insertion ordering has become a language guarantee starting from 3.7. It was guaranteed for cPython in 3.6. Just wanted to make sure this is not an issue.

Reply:
Thanks. I will push the change then.